Resources/Prototypes/Reagents/medicine.yml

```
@@ -220,10 +220,10 @@
           - !type:Temperature
             max: 213.0
           damage:
-            types:
-             Cellular: -4
-            groups:
-                Brute: 1
+            types: #WWDP edit start
+              Radiation: -4
+              Cellular: -4
+                   #WWDP edit end
```

```suggestion
            types:
              #WWDP edit start
              Radiation: -4
              Cellular: -4
              #WWDP edit end
```
Лучше покрывать марками с обоих сторон, а не добавлять поверх верхней и нижней строчки

---

теперь буду делать так